ubuntu2004/http/user-data

```diff
@@ -0,0 +1,44 @@
+#cloud-config
+autoinstall:
+  version: 1
+  apt:
+    geoip: true
+    preserve_sources_list: true
+#    primary:
+#    - arches: [amd64, i386]
+#      uri: http://de.archive.ubuntu.com/ubuntu
+#    - arches: [default]
+#      uri: http://ports.ubuntu.com/ubuntu-ports
+  packages:
+    - ubuntu-standard
```

Comment:
Did you find you needed to add all of the default packages the installer would need (like `ubuntu-standard`) if you set some of your own? This would be surprising to me if that's the case

Reply:
I added exactly all the same packages as Jeff had in the preseed. Careful ybout the "ubuntu-standard" package. That was just called "standard" in the preseed and caused a weird lookibg error instead of a simple "package 'standard' not found in available sources".

Reply:
Ah got it! Make sense.